src/hooks/api/write/useGetAnswers.ts

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from "@tanstack/react-query";
+
+import { api } from "@/api";
+
+type AnswerResponseType = {
+  answers: {
+    questionId: number;
+    questionType: string;
+    answerContent: string;
+  }[];
+};
+export const useGetAnswers = ({ spaceId, retrospectId }: { spaceId: number; retrospectId: number }) => {
+  const getQuestions = () => {
+    const res = api
+      .get(
+        `/space/${spaceId}/retrospect/${retrospectId}/answer/written
+`,
+      )
+      .then((res) => res.data as AnswerResponseType);
```

Comment:
시현이 PR 보고 알았는데, 이렇게 제네릭에 데이터 타입 지정해주면 타입 단언을 안해도 되더라구요 !
```suggestion
    const res = api
      .get<AnswerResponseType>(
        `/space/${spaceId}/retrospect/${retrospectId}/answer/written
`,
      )
      .then((res) => res.data);
```

Reply:
오호 계속 타입 단언으로 사용하고 있었는데, 제네릭 형태로 사용을 해봐야겠네요!

> 1. 제네릭으로 API 응답 타입을 명확히 지정하면 타입의 안정성이 높아진다.
> 2. 불필요한 타입 단언을 피해 가독성을 높일 수 있다.